ddsketch/store/store.go

```diff
@@ -54,6 +53,7 @@ type Store interface {
 	KeyAtRank(rank float64) int
 	MergeWith(store Store)
 	ToProto() *sketchpb.Store
+	EncodeProto(builder *sketchpb.StoreBuilder) // TODO: this can be private
```

Comment:
Why could this be private? Also, how big could be the impact for the library's users? Changing interfaces are breaking changes, but I'm not sure how do you handle this kind of changes and how often.

Beyond this, the PR looks fine.

Reply:
re: interface change

Whenever the users update their version of sketches-go, they'll need to add the new function
I think that's reasonable, and that's behind a version tag

re: `could this be private`

I'm not sure Lee's purpose
I'll wait for him @leeavital to reply

Reply:
It's because this method needs to be called from the top level package (`ddsketch`)

Reply:
I guess that this library may have a different way to handle breaking changes than dd-trace-go 😁 We are very wary of changing interfaces. But if this is your usual way, then it's fine.

re: `could this be private` I'd remove the comment as it seems than it's not possible to be private. Just to not introduce confusion in the future when somebody stumbles upon this.